specs/trs/tracked-resource-set.html

```diff
@@ -423,15 +423,28 @@ <h2>Base Segmentation</h2>
       <p>
         The number of Tracked Resources in the Base could be such that it is not reasonable to list them all in a single
         HTTP response. In this case,
-        <span class="conformance"
-          >the Base MAY be segmented into separate resources; there can be any number of such Base segments, each one
-          listing a subset of all Tracked Resources</span
-        >.
+        <span class="conformance">
+          the Base MAY be segmented into separate resources; there can be any number of such Base segments, each one
+          listing a subset of all Tracked Resources.</span
+        >
+      </p>
+      <p class="conformance">
+        If base paging is implemented, the server MAY respond with a 30x redirect message, directing the Client to the
```

Comment:
Two MAYs don't make a MUST this way. A "MUST x or y" would be better.

I also thought that we base our standard on web principles and HTTP already allows 30x redirects anywhere, so this is a bit redundant.

Finally, we are not specifying which 30x should be used, so I guess the semantics mirror HTTP in this regard. For example, see https://www.w3.org/TR/ldp-paging/#h-ldpp-client-nofollow-303